src/identity/oauth.test.ts

```diff
@@ -36,6 +36,12 @@ describe('OAuthProvider', () => {
     mockedExec.mockReset();
   });
 
+  const mockedConsoleLog = jest.spyOn(console, 'log').mockImplementation();
+
+  afterAll(() => {
+    mockedConsoleLog.mockRestore();
+  });
```

Comment:
If we send the "Your browser will open..." message to stderr we can ditch this bit -- I've already got `console.error` stubbed out [here](https://github.com/sigstore/sigstore-js/blob/main/jest.setup.js#L16).